docker/genomicsdb_builder/usr/bin/build_genomicsdb

```diff
@@ -0,0 +1,99 @@
+#! /bin/bash
+# The MIT License (MIT)
+# Copyright (c) 2017 Intel Corporation
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+if [ ! -d /output ]; then
+  echo 'can not find /output/ exit...'
+  return 2
+fi
+export BUILD_ROOT=${HOME}/build_src
+export PROTOBUF_LIBRARY=$BUILD_ROOT/protobuf_build
+echo "build_home=$build_home, PROTOBUF_LIBRARY=$PROTOBUF_LIBRARY, GenomicsDB_HOME=$GenomicsDB_HOME"
+
+protobuf_to_dir=/output
+genomicsdb_to_dir=/output
+
+# make protobuf
+build_proto_buf() {
+  echo "+++ Building protobuf at ${PROTOBUF_LIBRARY}..."
```

Comment:
the message is wrong.  you're not building protobuf under $BUILD_ROOT/protobuf_build, you're building it under /output